scripts/generate_dump

```diff
@@ -771,14 +808,17 @@ collect_mellanox() {
     done
 
     ${CMD_PREFIX}rm -rf $sai_dump_folder
-    ${CMD_PREFIX}docker exec -it syncd rm -rf $sai_dump_folder
+    ${CMD_PREFIX}docker exec -t syncd rm -rf $sai_dump_folder
     
     # Save SDK error dumps
     local sdk_dump_path=`${CMD_PREFIX}docker exec syncd cat /tmp/sai.profile|grep "SAI_DUMP_STORE_PATH"|cut -d = -f2`
-    if [[ $sdk_dump_path ]]; then
+    if [[ -d $sdk_dump_path ]]; then
         copy_from_docker syncd $sdk_dump_path  /tmp/sdk-dumps
         for file in $(find /tmp/sdk-dumps); do
```

Comment:
>find [](http://example.com/codeflow?start=22&length=4)

You may fix the issue by option `-type f`

Reply:
Yeah, Will do that

Reply:
Handled